src/components/general/reportOverview.js

```diff
@@ -1,29 +1,32 @@
-import React from 'react'
-import { View, Text, TouchableHighlight, StyleSheet } from 'react-native'
-import style from '../../library/styles/styleReportOverview'
-
+import React from 'react';
+import {View, Text, TouchableHighlight} from 'react-native';
+import style from '../../library/styles/styleReportOverview';
 
 export default function ReportOverview(props) {
-    return (
-        <View style={style.reportDetails}>
-            <View>
-                <Text>🤬</Text>
-            </View>
-            <View >
-                <Text> {props.userName} </Text>
-                <Text> {props.reportType} </Text>
-            </View>
-            <View>
-                <Text>{props.reportLocation}</Text>
-                <View style={style.reportInteractions}>
-                    <TouchableHighlight onPress={()=>{console.log('LIKE');}}>
-                        <Text>👍</Text>
-                    </TouchableHighlight>
-                    <TouchableHighlight onPress={()=>{console.log('REPORT DETS');}}>
-                        <Text>👁️</Text>
-                    </TouchableHighlight>
-                </View>
-            </View>
+  return (
+    <View style={style.reportDetails}>
+      <View>
+        <Text>🤬</Text>
+      </View>
+      <View>
+        <Text> {props.userName} </Text>
+        <Text> {props.reportType} </Text>
+      </View>
+      <View>
+        <Text>{props.reportLocation}</Text>
+        <View style={style.reportInteractions}>
+          <TouchableHighlight
+            onPress={() => {
+              console.log('LIKE');
```

Comment:
It is not recommended to leave the console.log () on the main branch. 

You can make a function to understand what you want that element to do, example:

```
export default function ReportOverview(props) {
  const [count, setCount] = useState(0);
  const onPress = () => setCount(prevCount => prevCount + 1);
  return (
    <View>
        <Text>Count: {count}</Text>
    </View>
  );
};
```